NFIQ2/NFIQ2Algorithm/src/prediction/RandomForestML.cpp

```diff
@@ -123,6 +123,58 @@ NFIQ2::Prediction::RandomForestML::initModule(const std::string &fileName,
 	return hash;
 }
 
+#ifdef __ANDROID__
+std::string
+NFIQ2::Prediction::RandomForestML::initModule(AAssetManager* assets,
+    const std::string &fileName,
+    const std::string &fileHash)
+{
+	if( assets == nullptr )
+	{
+		throw NFIQ2::Exception(NFIQ2::ErrorCode::InvalidConfiguration,
+		    "The trained network could not be initialized!");
```

Comment:
Communicate a more descriptive error.

```suggestion
		throw NFIQ2::Exception(NFIQ2::ErrorCode::InvalidConfiguration,
		    "Prediction::RandomForestML::initModule: AAssetManager is "
		    "null");
```